packages/labs/src/query-list/queryList.tsx

```diff
@@ -217,6 +217,12 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         }
     }
 
+    private handlingEmptyListCorrectly() {
+        const emptyFilteredList = this.state.filteredItems.length === 0;
+        const activeItemIsUndefined = this.props.activeItem === undefined;
+        return emptyFilteredList && activeItemIsUndefined;
```

Comment:
I think this honestly might make more sense to inline, above, since it's just in one place, and the function name is not particularly helpful. Plus the logic is harder to follow like this, I think.